backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -65,6 +65,10 @@ body {
   @include clearfix;
 }
 
+.content-under-header {
+  padding-top: 89px;
```

Comment:
What does this value represent? If it's the header's height we should save that as a value: `$height-header` and use it here and on the header.